python/paddle/tensor/linalg.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 import numpy as np
-from ..fluid.layer_helper import LayerHelper
+from ..framework import LayerHelper
 from ..framework import _varbase_creator, _dygraph_tracer
 from ..fluid.data_feeder import check_variable_and_dtype, check_type, check_dtype
 from ..static import Variable
```

Comment:
L21 below: from ..fluid.layers import transpose, cast  # noqa: F401
can we delete transpose(not use below) and use paddle.cast instead of fluid.layers.cast

Reply:
这个文件已在pr#40034中修改，这里只单独修改了LayerHelper